app/controllers/benefits/accommodation/LivingAccommodationBenefitAmountController.scala

```diff
@@ -50,13 +50,11 @@ class LivingAccommodationBenefitAmountController @Inject()(authAction: Authorise
 
   def show(taxYear: Int, employmentId: String): Action[AnyContent] = authAction.async { implicit request =>
     inYearAction.notInYear(taxYear) {
-      employmentSessionService.getAndHandle(taxYear, employmentId) { (optCya, prior) =>
+      employmentSessionService.getAndHandle(taxYear, employmentId) { (optCya, _) =>
         redirectBasedOnCurrentAnswers(taxYear, employmentId, optCya, EmploymentBenefitsType)(redirects(_, taxYear, employmentId)) { cya =>
           val cyaAmount = cya.employment.employmentBenefits.flatMap(_.accommodationRelocationModel.flatMap(_.accommodation))
-          val form = fillFormFromPriorAndCYA(formsProvider.livingAccommodationAmountForm(request.user.isAgent), prior, cyaAmount, employmentId)(
-            employment => employment.employmentBenefits.flatMap(_.benefits.flatMap(_.accommodation))
-          )
-          Future(Ok(livingAccommodationBenefitAmountView(taxYear, form, cyaAmount, employmentId)))
+          val form = fillForm(formsProvider.livingAccommodationAmountForm(request.user.isAgent), cyaAmount)
+          Future(Ok(pageView(taxYear, form, cyaAmount, employmentId)))
```

Comment:
Do we need the `cyaAmount` here? We are filling it in the line above, right?

Reply:
You are right we don't. I will change.